annotations/builder/src/main/java/io/sundr/builder/internal/functions/ClazzAs.java

```diff
@@ -214,12 +217,15 @@ public TypeDef apply(TypeDef item) {
                 final boolean isSet = TypeUtils.isSet(property.getTypeRef());
                 final boolean isList = TypeUtils.isList(property.getTypeRef());
                 final boolean isMap = TypeUtils.isMap(property.getTypeRef());
-                final boolean isCollection = isSet || isList;
                 final boolean isAbstract = isAbstract(unwrapped);
+                final boolean isOptionalInt = TypeUtils.isOptionalInt(property.getTypeRef());
```

Comment:
Same. It seem that we just want to know if its optional.

Reply:
Updated.